ios/PresentationLayer/UIToolkit/Sources/UIToolkit/BaseViews/SwiftUI/Image/RemoteImage.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Created by David Kadlček on 12.05.2023
+//  Copyright © 2023 Matee. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+public struct RemoteImage: View {
+    
+    @State private var image: Image?
+    private let stringURL: String?
+    private let defaultImage: Image
+    private let contentMode: ContentMode
+    private let failureMessage: String?
+    
+    public init(
+        stringURL: String?,
+        defaultImage: Image,
+        contentMode: ContentMode = .fit,
+        failureMessage: String? = nil
```

Comment:
Why is failure message passed via the initializer? 🤔 Is it not supposed to be a failure message from fetching the image?

Reply:
I just copied it from another project. This was because we got some error message, so we show it instead of an image. I can delete it for Devstack.